Source/hu_stuff.c

```diff
@@ -566,6 +566,10 @@ void HU_Start(void)
   // [FG] fix crash when gamemap is not initialized
   if (gamestate == GS_LEVEL && gamemap > 0)
   {
+  // initialize the map title widget with the generic map lump name
+  s = MAPNAME(gameepisode, gamemap);
```

Comment:
Please turn this into an `if...else` condition with braces around the original code and indentation for the new one.